source/Calamari.AzureScripting/Calamari.AzureScripting.csproj

```diff
@@ -8,21 +8,25 @@
         <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
         <RuntimeIdentifiers>win-x64;linux-x64;osx-x64;linux-arm;linux-arm64</RuntimeIdentifiers>
         <IsPackable>true</IsPackable>
+        <ValidateExecutableReferencesMatchSelfContained>false</ValidateExecutableReferencesMatchSelfContained>
```

Comment:
This is to make this project able to include another project that outputs a self-contained exe, am I understanding it correctly?

Reply:
Yup, that's correct.